packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -310,9 +317,11 @@ export default class ComboBox extends React.Component {
               })}>
               <input
                 className={`${prefix}--text-input`}
-                aria-label={ariaLabel}
+                aria-labelledby={comboBoxA11yId}
```

Comment:
This was surprising to me, my first thought was that `ListBox.Field` would provide the label for the input

Reply:
It seemed like the aria-label should be on the combobox container element which I took to be the ListBox but I might be wrong about that. Would the listBox.Field be considered the container?

Reply:
Lol ignore me, I thought `ListBox.Field` had the `<label>` for some reason 🤦‍♂️